erpnext/erpnext_integrations/connectors/shopify_connection.py

```diff
@@ -246,6 +251,7 @@ def update_taxes_with_shipping_lines(taxes, shipping_lines, shopify_settings):
 	return taxes
 
 def get_tax_account_head(tax):
+	return
```

Comment:
why?

Reply:
I had added it while testing, forgot to remove it